coldfront/core/allocation/templates/allocation/allocation_cluster_account_request_list_table.html

```diff
@@ -0,0 +1,130 @@
+<table class="table table-sm">
+  <thead>
+  <tr>
+    <th scope="col">
+      #
+      <a href="?order_by=id&direction=asc&{{filter_parameters}}"><i class="fas fa-sort-up"></i></a>
+      <a href="?order_by=id&direction=des&{{filter_parameters}}"><i class="fas fa-sort-down"></i></a>
+    </th>
+    <th scope="col">
+      Date Requested/<br>Last Modified
+      <a href="?order_by=created&direction=asc&{{filter_parameters}}"><i class="fas fa-sort-up"></i></a>
+      <a href="?order_by=created&direction=des&{{filter_parameters}}"><i class="fas fa-sort-down"></i></a>
+    </th>
+    <th scope="col">
+      User Email
+      <a href="?order_by=allocation_user__user__email&direction=asc&{{filter_parameters}}"><i class="fas fa-sort-up"></i></a>
+      <a href="?order_by=allocation_user__user__email&direction=des&{{filter_parameters}}"><i class="fas fa-sort-down"></i></a>
+    </th>
+    <th scope="col">
+      Cluster Username
+      <a href="?order_by=allocation_user__user__username&direction=asc&{{filter_parameters}}"><i class="fas fa-sort-up"></i></a>
+      <a href="?order_by=allocation_user__user__username&direction=des&{{filter_parameters}}"><i class="fas fa-sort-down"></i></a>
+    </th>
+    <th scope="col">
+      Project
+      <a href="?order_by=allocation__project__name&direction=asc&{{filter_parameters}}"><i class="fas fa-sort-up"></i></a>
+      <a href="?order_by=allocation__project__name=des&{{filter_parameters}}"><i class="fas fa-sort-down"></i></a>
+    </th>
+    <th scope="col">
+      Allocation
+    </th>
+    <th scope="col">
+      Status
+    </th>
+    {% if user.is_superuser and request_filter == 'pending' %}
```

Comment:
Correct me if I'm wrong, but I think you're relying on `request_filter` not being passed to the request hub view to hide them.

It'd be clearer to explicitly pass a separate boolean variable `actions_visible` to the `include` tag:

```
{% with queryset=cluster_account_list actions_visible=false %}
  {% include 'allocation/allocation_cluster_account_request_list_table.html' %}
{% endwith %}
```

In the main page:

```
{% with queryset=cluster_account_list actions_visible=request_filter == 'pending' %}
  {% include 'allocation/allocation_cluster_account_request_list_table.html' %}
{% endwith %}
```

I'm not sure if you can set a variable to an expression, so you might need to set `actions_variable` from the view.

Reply:
I did this for project removal requests as well. We are passing actions_visisble to all 7 table templates now, even though 5 of them do not use the variable